x-pack/legacy/plugins/canvas/canvas_plugin_src/uis/arguments/axis_config/simple_template.tsx

```diff
@@ -4,13 +4,20 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React from 'react';
+import React, { SFC } from 'react';
```

Comment:
I see `FunctionComponent` used elsewhere in this PR, which is I think is the preferred name.